### PR TITLE
[MontessoriSchool] Fix category

### PR DESCRIPTION
**Pull request by karen-kerr**
{'atp/brand/Montessori School': 26,
 'atp/brand_wikidata/Q114840336': 26,
 'atp/category/amenity/kindergarten': 26,
 'atp/field/country/from_reverse_geocoding': 26,
 'atp/field/email/missing': 26,
 'atp/field/image/missing': 26,
 'atp/field/opening_hours/missing': 26,
 'atp/field/operator/missing': 26,
 'atp/field/operator_wikidata/missing': 26,
 'atp/field/phone/missing': 26,
 'atp/field/twitter/missing': 26,
 'atp/field/website/missing': 26,
 'atp/nsi/perfect_match': 26,
 'downloader/request_bytes': 5350,
 'downloader/request_count': 9,
 'downloader/request_method_count/GET': 9,
 'downloader/response_bytes': 419940,
 'downloader/response_count': 9,
 'downloader/response_status_count/200': 9,
 'elapsed_time_seconds': 12.097382,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2024, 1, 24, 10, 53, 12, 776131, tzinfo=datetime.timezone.utc),
 'item_scraped_count': 26,
 'log_count/DEBUG': 46,
 'log_count/INFO': 9,
 'memusage/max': 137023488,
 'memusage/startup': 137023488,
 'request_depth_max': 1,
 'response_received_count': 9,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 8,
 'scheduler/dequeued/memory': 8,
 'scheduler/enqueued': 8,
 'scheduler/enqueued/memory': 8,
 'start_time': datetime.datetime(2024, 1, 24, 10, 53, 0, 678749, tzinfo=datetime.timezone.utc)}

### CODE REVIEW

File: locations/spiders/montessori_school.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 class MontessoriSchoolSpider(scrapy.Spider):
     name = "montessori_school"
-    item_attributes = {"brand": "Montessori School"}
+    item_attributes = {"brand": "Montessori School", "brand_wikidata": "Q114840336"}
```
Comment (CloCkWeRX) [inline locations/spiders/montessori_school.py:10]: https://www.wikidata.org/wiki/Q114840336 mentions "Guidepost Montessori", while this scraper is for  https://www.montessori.com/ ©2024 La Petite Academy, LLC - are they the same entity?

Comment (karen-kerr) [reply locations/spiders/montessori_school.py:10]: Thanks, good catch! I've dug a bit further into this. 
Guidepost Montessori seems to be a commercial version/branch of Montessori schools. Their Q code is matching into NSI with amenity/kindergarten.
Montessori schools are up to age 18, so I've set category to be amenity/school.

Comment (karen-kerr) [reply locations/spiders/montessori_school.py]: {'atp/brand/Montessori School': 26,
 'atp/brand_wikidata/Q56280103': 26,
 'atp/category/amenity/school': 26,
 'atp/field/country/from_reverse_geocoding': 26,
 'atp/field/email/missing': 26,
 'atp/field/image/missing': 26,
 'atp/field/opening_hours/missing': 26,
 'atp/field/operator/missing': 26,
 'atp/field/operator_wikidata/missing': 26,
 'atp/field/phone/missing': 26,
 'atp/field/twitter/missing': 26,
 'atp/field/website/missing': 26,
 'atp/nsi/brand_missing': 26,
 'downloader/request_bytes': 5350,
 'downloader/request_count': 9,
 'downloader/request_method_count/GET': 9,
 'downloader/response_bytes': 419942,
 'downloader/response_count': 9,
 'downloader/response_status_count/200': 9,
 'elapsed_time_seconds': 12.891921,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2024, 1, 30, 14, 12, 48, 661351, tzinfo=datetime.timezone.utc),
 'item_scraped_count': 26,
 'log_count/DEBUG': 46,
 'log_count/INFO': 9,
 'memusage/max': 136282112,
 'memusage/startup': 136282112,
 'request_depth_max': 1,
 'response_received_count': 9,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 8,
 'scheduler/dequeued/memory': 8,
 'scheduler/enqueued': 8,
 'scheduler/enqueued/memory': 8,
 'start_time': datetime.datetime(2024, 1, 30, 14, 12, 35, 769430, tzinfo=datetime.timezone.utc)}

Comment (karen-kerr) [reply locations/spiders/montessori_school.py]: That's not the correct wikidata code either, so I've removed it.
{'atp/brand/Montessori School': 26,
 'atp/category/amenity/school': 26,
 'atp/field/brand_wikidata/missing': 26,
 'atp/field/country/from_reverse_geocoding': 26,
 'atp/field/email/missing': 26,
 'atp/field/image/missing': 26,
 'atp/field/opening_hours/missing': 26,
 'atp/field/operator/missing': 26,
 'atp/field/operator_wikidata/missing': 26,
 'atp/field/phone/missing': 26,
 'atp/field/twitter/missing': 26,
 'atp/field/website/missing': 26,
 'downloader/request_bytes': 5350,
 'downloader/request_count': 9,
 'downloader/request_method_count/GET': 9,
 'downloader/response_bytes': 419942,
 'downloader/response_count': 9,
 'downloader/response_status_count/200': 9,
 'elapsed_time_seconds': 12.934459,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2024, 1, 30, 14, 21, 22, 477613, tzinfo=datetime.timezone.utc),
 'item_scraped_count': 26,
 'log_count/DEBUG': 36,
 'log_count/INFO': 9,
 'memusage/max': 136757248,
 'memusage/startup': 136486912,
 'request_depth_max': 1,
 'response_received_count': 9,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 8,
 'scheduler/dequeued/memory': 8,
 'scheduler/enqueued': 8,
 'scheduler/enqueued/memory': 8,
 'start_time': datetime.datetime(2024, 1, 30, 14, 21, 9, 543154, tzinfo=datetime.timezone.utc)}

**Pull request merged**